src/test/java/com/squareup/javapoet/MethodSpecTest.java

```diff
@@ -302,4 +303,16 @@ static void staticMethod() {
       assertThat(e.getMessage()).isEqualTo("modifiers == null");
     }
   }
+
+  @Test public void modifyMethodName() {
+    MethodSpec methodSpec = MethodSpec.methodBuilder("initialMethod")
+            .build()
```

Comment:
Continuation indents should be 4 spaces

Reply:
Fixed fe3f339c168ddca267c2e6ae0aad095b455d8c5f